project/plugins.sbt

```diff
@@ -3,20 +3,14 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-// Try to remove when sbt 1.8.0+ and scoverage is 2.0.7+
-ThisBuild / libraryDependencySchemes ++= Seq(
-  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
-)
-
-addSbtPlugin("com.typesafe.play"   % "sbt-plugin"               % "2.8.21")
-addSbtPlugin("io.github.irundaia"  % "sbt-sassify"              % "1.5.2")
-addSbtPlugin("org.scalastyle"      % "scalastyle-sbt-plugin"    % "1.0.0")
-addSbtPlugin("org.scoverage"       % "sbt-scoverage"            % "2.0.11")
-addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"           % "3.20.0")
-addSbtPlugin("uk.gov.hmrc"         % "sbt-distributables"       % "2.5.0")
-addSbtPlugin("com.timushev.sbt"    % "sbt-updates"              % "0.6.4")
-addSbtPlugin("org.scalameta"       % "sbt-scalafmt"             % "2.5.2")
-addSbtPlugin("com.typesafe.sbt"    % "sbt-digest"               % "1.1.4")
-addSbtPlugin("net.ground5hark.sbt" % "sbt-concat"               % "0.2.0")
-addSbtPlugin("uk.gov.hmrc"         % "sbt-accessibility-linter" % "0.37.0")
+addSbtPlugin("org.playframework"  % "sbt-plugin"               % "3.0.2")
+addSbtPlugin("io.github.irundaia" % "sbt-sassify"              % "1.5.2")
+addSbtPlugin("org.scalastyle"     % "scalastyle-sbt-plugin"    % "1.0.0" exclude ("org.scala-lang.modules", "scala-xml_2.12"))
+addSbtPlugin("org.scoverage"      % "sbt-scoverage"            % "2.0.11")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-auto-build"           % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-distributables"       % "2.5.0")
+addSbtPlugin("com.timushev.sbt"   % "sbt-updates"              % "0.6.4")
+addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.5.2")
+addSbtPlugin("com.github.sbt"     % "sbt-digest"               % "2.0.0")
+addSbtPlugin("com.github.sbt"     % "sbt-concat"               % "1.0.0")
+addSbtPlugin("uk.gov.hmrc"        % "sbt-accessibility-linter" % "0.39.0")
```

Comment:
When you run the linter on this version for play 3.0 you will see that `FrontendAccessibilitySpec` can't pick up the CSRF token for the form on the view `search.scala.html`. It seems that this view is still using an old format 
```
  @helper.form(action = routes.SearchController.get()) {
    @helper.CSRF.formField
    @searchInput(form)
  }
```

We now should use `import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF` on the template, as it handles the `formField` value.  See here https://github.com/hmrc/play-frontend-hmrc/blob/72218146aead72d58c846d06d0b7e6321222d3f3/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/FormWithCSRF.scala.html#L28